tests/all_sh_components.txt

```diff
@@ -206,3 +206,15 @@ component_test_cmake_shared () {
     cd "$TF_PSA_CRYPTO_ROOT_DIR"
     rm -rf "$OUT_OF_SOURCE_DIR"
 }
+
+component_check_code_style () {
+    msg "Check C code style"
+    ./scripts/code_style.py
```

Comment:
I've tried to run it and it seems we miss `check-generated-files.sh`.

Reply:
Hmm, ok not sure why. Will investigate.

Reply:
As discussed, I misunderstood the meaning of the initial comment. I have added `check-generated-files.sh` to be copied in `psa_crypto.py`.